src/_includes/de/mission_statement.njk

```diff
@@ -28,12 +28,12 @@
 
 <div class="columns is-desktop">
   <div class="column">
-    <h3 class="is-size-3 mb-4">Code of Conduct</h3>
+    <h3 class="is-size-3 mb-4">Verhaltenskodex</h3>
     <p>
-      Do you coach other juggers, promote your club locally, organize tournaments, or otherwise serve as a leader in your local jugger club?  <strong>The IJC Code of Conduct can help you create an safe, enjoyable and inclusive environment</strong> for the sport of jugger.
+      Trainierst du andere Jugger, förderst du deinen Club vor Ort, organisierst du Turniere oder bekleidest du auf andere Weise eine Führungsposition in deiner lokalen Jugger-Organisation? Der IJC-Verhaltenskodex bietet einen nützlichen Ausgangspunkt, um <strong>ein sicheres, angenehmes und integratives Umfeld für den Jugger-Sport zu schaffen</strong>.
     </p>
     <p class="is-size-3">
-      <a href="https://docs.google.com/document/d/1PA8wOP_-ITgONZFS6FuFjjPXXVSUNqZY6RtcvfJF268/edit">IJC Code of Conduct</a>
+      <a href="https://docs.google.com/document/d/1PA8wOP_-ITgONZFS6FuFjjPXXVSUNqZY6RtcvfJF268/edit">IJC-Verhaltenskodex</a>
     </p>
```

Comment:
For the quoted blurb in this section, we unfortunately don't have a translation.  (This does make it a _bit_ weird that we've translated the mission statement and not this, but my German is wayyyyyyy too basic to take a stab at translating this myself.)